getting-started-ng/success.md

```diff
@@ -1,7 +1,12 @@
-Now that you’ve learned the NativeScript basics, you’re ready to move on to our next tutorial, which has you built a complete iOS and Android app from scratch.
+Now that you’ve learned the NativeScript basics, you’re ready to move on to our next tutorial, which has you built a complete iOS and Android app from scratch. To continue you can:
```

Comment:
"(...), which has you built a (...)" doesn't sound right to me. Consider replacing it with:
1. "(...) in which you get to build a complete (...)" 
2. "(...) in which you'll see how to build a complete (...)"